packages/modules/src/modules/epics/ContributionsEpicChoiceCards.tsx

```diff
@@ -91,6 +121,80 @@ export const ContributionsEpicChoiceCards: React.FC<EpicChoiceCardProps> = ({
         }[selection.frequency];
     };
 
+    const generateChoiceCardAmountsButtons = () => {
+        const requiredAmounts = amountsTestVariant[selection.frequency].amounts;
+        const defaultAmount = amountsTestVariant[selection.frequency].defaultAmount;
+        console.log(defaultAmount, requiredAmounts);
+
+        // Something is wrong with the data
+        if (!Array.isArray(requiredAmounts) || !requiredAmounts.length) {
+            return <ChoiceCard value="third" label="Other" id="third" checked={true} />;
+        }
+
+        // hideChooseYourAmount === true
```

Comment:
I think you can make this a lot shorter if you create a `ChoiceCardAmount` component, e.g.:
https://github.com/guardian/support-dotcom-components/pull/835/files

Reply:
Merged